zkevm-circuits/src/evm_circuit/execution/timestamp.rs

```diff
@@ -28,14 +27,14 @@ impl<F: FieldExt> ExecutionGadget<F> for TimestampGadget<F> {
     const EXECUTION_STATE: ExecutionState = ExecutionState::TIMESTAMP;
 
     fn configure(cb: &mut ConstraintBuilder<F>) -> Self {
-        let timestamp = cb.query_rlc();
+        let timestamp = cb.query_cell();
         cb.stack_push(timestamp.expr());
 
         // Lookup block table with timestamp
         cb.block_lookup(
-            BlockContextFieldTag::Time.expr(),
+            BlockContextFieldTag::Timestamp.expr(),
             None,
-            from_bytes::expr(&timestamp.cells),
+            timestamp.expr(),
```

Comment:
This means that the Timestamp value in BlockContext is RLC encoded right?  Why is that?  If this is the way, the spec must also be updated.

Reply:
Ah, I just realized the spec has been updated in appliedzkp/zkevm-specs#105 (`timestamp` is an acual value instead of `RLC` in `block_table`)..

Reply:
I think it makes sense to keep values that fit inside the field (like timestamps) without RLC encoding in the tables.  We already do this for addresses.
Do you think it would be better to keep the timestamp RLC encoded in the block table?

Reply:
I see!  I like EIP1985 and I also think we can follow it.  It will make thinks a bit easier, and it's compatible with the way geth works.

Reply:
Undo the change in [`9bbbac6`](https://github.com/appliedzkp/zkevm-circuits/pull/292/commits/9bbbac6977e6e959305abc933a0bd24614694e37). Will file a PR on spec about EIP 1985 later.